pkg/operator/staticpod/certsyncpod/certsync_cmd.go

```diff
@@ -90,7 +88,11 @@ func (o *CertSyncControllerOptions) Run() error {
 	if err != nil {
 		return err
 	}
+
+	// start everything. Informers start after they have been requested.
 	go controller.Run(1, stopCh)
+	go observer.Run(stopCh)
+	go kubeInformers.Start(stopCh)
```

Comment:
nit: I think you don't have to start Kube informers in a separate "thread" because all informers will be run in a dedicated goroutine, see https://github.com/openshift/library-go/blob/5dbeda57c0d97b335dd66bff593d0f08bc8cadf3/vendor/k8s.io/client-go/informers/factory.go#L132

maybe we could change the name of the method to `StartAsync` :)